### ISSUE TITLE
Enforce Draft Pull requests ?

### ISSUE THREAD

**Issue by sadortun** [parse-community/parse-issue-bot#17]
Hi !! 

Just an idea !  There is quite a lot of PRs (including mine, haha) that exists, but are really not ready to merge yet. It would be nice if this bot would ask/enforce user to create `draft` pull requests when its not ready to merge. It would declutter the PR list.

Have a good weekend,
Samuel

**Comment by mtrezza** [comment]
Good idea! This bot is about to be replaced by a GitHub app. That is why there is no enforcement whatsoever for PRs at this point.
